src/main/java/org/folio/circulation/domain/representations/AccountRepresentation.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.circulation.domain.representations;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.FeeFine;
+import org.folio.circulation.domain.FeeFineOwner;
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.support.JsonPropertyWriter;
+
+import io.vertx.core.json.JsonObject;
+
+public class AccountRepresentation extends JsonObject {
+
+  public AccountRepresentation(Loan loan, Item item, FeeFineOwner feeFineOwner,
+                               FeeFine feeFine, Double amount) {
```

Comment:
Please use the standard line indentation rather than horizontal alignment (which is harder to maintain and wastes horizontal space)